packages/block-editor/src/components/spacing-sizes-control/style.scss

```diff
@@ -17,21 +17,34 @@
 		align-self: center;
 	}
 
+	.components-base-control__label {
+		margin-bottom: 0;
+		height: 16px;
+	}
+
 	.components-spacing-sizes-control__side-labels {
 		grid-column: 1 / 1;
-		min-height: 30px;
 		justify-content: left;
+		height: 16px;
+		margin-top: 12px;
```

Comment:
`$grid-unit-15`.

Reply:
yeh, probably makes sense to use the vars for these